openshift/templates/pims-api/deploy.yaml

```diff
@@ -156,6 +156,22 @@ objects:
     spec:
       strategy:
         type: Rolling
+        rollingParams:
+          intervalSeconds: 1
+          maxSurge: 25%
+          maxUnavailable: 25%
+          pre:
+            execNewPod:
+              command:
+                - bash
+                - "-c"
+                - >-
+                  dotnet ef database update --project=api --context=pimsContext
```

Comment:
Does this pick up all the configuration for the `api`?

Reply:
it ran as I expected in my simple test, not sure what else I can do here for validation.